### PR TITLE
Bump Lepton EDA version

### PR DESCRIPTION
**Pull request by vzh**

### CODE REVIEW

**Comment by graahnul-grom** [comment]
@vzh Should I make a PR for this (will it conflict with #283?)? Or if you are about to merge #283, maybe you could include that fix in it?

**Comment by vzh** [comment]
@graahnul-grom, thank you for the review!
> Should we hasten to make a new release just today?

If I do not do it now, I don't know when I'll be able to do it at all :-) Let's work iteratively. For me, large hunks of code are difficult to describe, I spend too much time on it (good PR descriptions do help a bit ;-)). Besides, distributions accept new versions as we release them, so we could have more feed back from them.
Though I'm not in a hurry (the date is easily to fix), so I'll try to review PRs you mentioned here first.

**Comment by graahnul-grom** [comment]
@vzh Thank you. I thought that 10/12 is a special day for you. :)

File: configure.ac [hunk 1]
```diff
@@ -183,7 +183,7 @@ AX_DATA_DIRS
 # Where should PCB footprints be searched for?
 AX_PCB_DIRS
 # Set up liblepton with the correct ld version number
-AX_LIBLEPTON([2:0:0])
+AX_LIBLEPTON([3:0:0])
```
[the remark this answers is not in the document]
Comment (vzh) [reply configure.ac]: Please let's do it in the next release. #294 breaks nothing, so we can just update the lib revision.

File: NEWS.md [hunk 2]
```diff
@@ -4,9 +4,96 @@ Recent Changes in Lepton EDA
...
+  ensure compatibility with legacy geda backends.
+
+- `schematic-packages` and `schematic-non-unique-packages` fields
+  have been removed from the `<schematic>` record since they are
+  derivatives from its other structures. Two substitution
+  procedures retrieving those data, `schematic-package-names` and
+  `schematic-non-unique-package-names`, have been provided to
+  provide the same functionality.
+
+- Symbols and schematics in `examples/vams` have been fixed.
```
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: As I said before, I have a local branch on `vams`, some excerpts from which I'd like to apply with new netlister features in the next release. Please let's wait with this, too.
In two words, the example is about using `gnetlist` executable through `gschem` menus. I hope we could do it better way some day.

File: NEWS.md [hunk 3]
```diff
@@ -4,9 +4,96 @@ Recent Changes in Lepton EDA
 This file documents important user-visible changes in Lepton EDA.  For
 more information, please consult `git log` history.
 
+Notable changes in Lepton EDA 1.9.7
+-----------------------------------
+
+### Changes in `lepton-schematic`:
```
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: I don't think this change is a **notable** one for our users :-) There is no point to copy the full `git log` history to NEWS.

File: NEWS.md [hunk 4]
```diff
@@ -4,9 +4,96 @@ Recent Changes in Lepton EDA
...
+
+- A new action, `edit-object-properties`, has been added. It is a
+  replacement for four legacy actions which opened the same
+  dialog: `edit-color`, `edit-linetype`, `edit-filltype`,
+  `edit-pin-type`. The action is now bound to the shortcut
+  <kbd>E</kbd>-<kbd>O</kbd>.
+
+- Fixed icon name for "Down Symbol" entry in pop-up menu.
```
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: This one is a good catch. I'll fix it.

File: NEWS.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: OK, thanks.

File: NEWS.md [hunk 5]
```diff
@@ -4,9 +4,96 @@ Recent Changes in Lepton EDA
...
+    Show/Hide Hidden Text', has been added while retaining the
+    legacy one, <kbd>E</kbd>-<kbd>N</kbd>.
+  - Shortcut for 'Help → Find Component Documentation' has been
+    changed to <kbd>H</kbd>-<kbd>O</kbd>.
+  - Shortcut for 'Options → Options' has been changed to
+    <kbd>O</kbd>-<kbd>O</kbd>.
+  - Shortcut for 'Attributes → Find Text' has been changed to
+    <kbd>T</kbd>-<kbd>F</kbd>.
+
+### Changes in `lepton-netlist`:
```
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: Again, this adds no functionality/performance, so there is no point to mention it.

File: NEWS.md [hunk 6]
```diff
@@ -4,9 +4,96 @@ Recent Changes in Lepton EDA
 This file documents important user-visible changes in Lepton EDA.  For
 more information, please consult `git log` history.
 
+Notable changes in Lepton EDA 1.9.7
+-----------------------------------
+
+### Changes in `lepton-schematic`:
+
+- A new action, `edit-object-properties`, has been added. It is a
```
[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: Dunno if it's worth it. We usually do this by default, and, other way around, inform the users if something was removed.

**Comment by vzh** [comment]
@graahnul-grom, as for #283, please make a PR with your fixes. I don't want to merge #283 it just now, it should go to the next release after this one. And I think I should postpone this one at least until tomorrow :-)

File: NEWS.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (graahnul-grom) [reply NEWS.md]: Then, remove `examples/vams` line from `NEWS`. There's no point to inform users about changes to thing that did not work and still does not work. These are, of course,not a **notable** changes. I would recommend to apply simple fixes to obvious bugs; when time comes to merge your`vams` branch, these changes are easy to revert.

[the remark this answers is not in the document]
Comment (vzh) [reply NEWS.md]: OK, thanks, I've removed it.

**Pull request merged**